spec/factories/rents.rb

```diff
@@ -0,0 +1,9 @@
+require 'faker'
+FactoryBot.define do
+  factory :rent do
+    user
+    book
+    from    Faker::Date.backward(10)
```

Comment:
`Faker::Date.backward(10)` should be in a block, `{ Faker::Date.backward(10) } `